source/administration/changelog.md

```diff
@@ -4,9 +4,269 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
-## Release v3.10.0
+## Release v4.0.0
 
-Release Date: June 16, 2017
+Release Date: July 16, 2017
+
+### Security Update
+
+- Mattermost v4.0.0 contains multiple security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 14 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
+
+### Highlights
+
+#### Native iOS and Android Apps
+- Second generation mobile apps released for [iOS](https://itunes.apple.com/us/app/mattermost/id1257222717?mt=8) and [Android](https://play.google.com/store/apps/details?id=com.mattermost.rn).
+- The apps are [EMM compatible starting with BlackBerry Dynamics](https://about.mattermost.com/mattermost-2nd-gen-mobile-apps-released-emm-compatible-starting-with-blackberry-dynamics/).
+
+#### Updated Web User Interface
+- Updated the appearance of channel header and channel sidebar in the web user interface.
+- Added a new default theme, "Mattermost". To try it, go to **Account Settings > Display > Theme**.
```

Comment:
So we have a "new default theme". The name per say didn't change

0/5 if we want to change the wording